server/validation.js

```diff
@@ -154,31 +155,56 @@ const EmployerFormSchema = yup.object().noUnknown('Unknown field in form').shape
   doesCertify: yup.boolean().typeError(errorMessage).required(errorMessage).test('is-true', errorMessage, (v) => v === true),
 });
 
+const EmployerSiteBatchSchema = yup.array().of(
+  yup.lazy((item, options) => {
+    const index = options.parent.indexOf(item);
+    return yup.object().noUnknown(`Unknown field in site data (index ${index})`).shape({
```

Comment:
Just a note on some fields that I think are required and not nullable:

- siteId
- siteName
- healthAuthority
- postalCode

I think too many other parts of the application hinge on these fields to allow them to be nullable. We need the site name in order to show it in the access approval dropdown, and we need the HA + postal code for region access controls and sorting/searching later